lib/matplotlib/tests/test_datetime.py

```diff
@@ -29,8 +29,20 @@ def test_arrow(self):
     @pytest.mark.xfail(reason="Test for axhline not written yet")
     @mpl.style.context("default")
     def test_axhline(self):
-        fig, ax = plt.subplots()
-        ax.axhline(...)
+        mpl.rcParams["date.converter"] = 'concise'
+        fig, ax = plt.subplots(layout='constrained')
+        birth_date = np.array([datetime.datetime(2020, 4, 10),
+                               datetime.datetime(2020, 5, 30),
+                               datetime.datetime(2020, 10, 12),
+                               datetime.datetime(2020, 11, 15)])
+        year_start = datetime.datetime(2020, 1, 1)
+        year_end = datetime.datetime(2020, 12, 31)
+        age = [21, 53, 20, 24]
+        ax.set_xlabel('Birth Date')
+        ax.set_ylabel('Age')
+        ax.set_ylim(bottom=year_start, top=year_end)
+        ax.bar(x=age, height=birth_date)
```

Comment:
I would say to keep this even more minimal and not have the bar graph (I think setting the ylim is important here)

If anything, I'd just do a `ax.plot`, but even that I think is not testing what we are looking to test here.